packages/mdc-tab/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@material/tab",
+  "description": "The Material Components for the web tab component",
+  "version": "0.33.0",
```

Comment:
this should be version 0.0.0 I think...then Lerna up the version number later.

I forget where the docs are, but somewhere there are docs saying how to create a new package.

Reply:
Dope, I'll check that out.